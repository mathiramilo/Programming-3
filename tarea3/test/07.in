# tres reinicios
leer_datos
1 10 20 30 40 50 60 .
leer_rendimiento
30 6 5 4 3 2 1 .
max_datos_procesados
planificacion_optima
Fin