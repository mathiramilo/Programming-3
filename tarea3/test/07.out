1>2>3>4>Max Datos Procesados: 81
5>Cantidad Datos Planificacion: 81
6>Fin.
